### DISCUSSION TITLE
is there a way to sort files by number outside of 1 to 9?

### DISCUSSION THREAD

**Discussion by NoahBranley** [community/community#73314] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

![image](https://github.com/community/community/assets/112705279/a63681af-767f-4a39-87d7-5efe56641883)
as you can see i want 1-10 on the bottom but because its (most likely) read as 1 1 0 its put second. how could i sort this nicely?

**Comment by b8zeek** [comment, marked as answer]
I guess you can rename the first 9 by adding a `0` after the dash; e.g. `exercise 1-01.c`, `exercise 1-02.c`, `exercise 1-03.c`, etc. That should solve the issue you have. 😸 

🍀

**Reply by NoahBranley** [reply]
thank you very much, i dont know why i didnt think of that!! life saver

**Comment by b8zeek** [comment]
One more thing, in case you'll have more than 99 files, you should start with `001`, `002`, `003`, etc. We do not want you to get `10` than `100`. 😅 

Cheers. 🍺 

🍀
